diff --git a/node_modules/prop-types/node_modules/react-is/cjs/react-is.development.js b/node_modules/prop-types/node_modules/react-is/cjs/react-is.development.js
index 8a80b76..4c629ec 100644
--- a/node_modules/prop-types/node_modules/react-is/cjs/react-is.development.js
+++ b/node_modules/prop-types/node_modules/react-is/cjs/react-is.development.js
@@ -6,13 +6,6 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
-'use strict';
-
-
-
-if (process.env.NODE_ENV !== "production") {
-  (function() {
 'use strict';
 
 // The Symbol used to tag the ReactElement-like types. If there is no native Symbol
@@ -177,5 +170,3 @@ exports.isStrictMode = isStrictMode;
 exports.isSuspense = isSuspense;
 exports.isValidElementType = isValidElementType;
 exports.typeOf = typeOf;
-  })();
-}
