diff --git a/node_modules/react-is/cjs/react-is.development.js b/node_modules/react-is/cjs/react-is.development.js
index 3126207..3f78261 100644
--- a/node_modules/react-is/cjs/react-is.development.js
+++ b/node_modules/react-is/cjs/react-is.development.js
@@ -11,8 +11,6 @@
 
 
 
-if (process.env.NODE_ENV !== "production") {
-  (function() {
 'use strict';
 
 // ATTENTION
@@ -207,5 +205,4 @@ exports.isStrictMode = isStrictMode;
 exports.isSuspense = isSuspense;
 exports.isValidElementType = isValidElementType;
 exports.typeOf = typeOf;
-  })();
-}
+  
